apps/minifront/src/components/shared/number-input/use-wheel-prevent.ts

```diff
@@ -15,12 +15,10 @@ export const useWheelPrevent = () => {
   };
 
   useEffect(() => {
-    inputRef.current?.addEventListener('wheel', onWheel);
-
-    return () => {
-      inputRef.current?.removeEventListener('wheel', onWheel);
-    };
-  }, []);
+    const ac = new AbortController();
+    inputRef.current?.addEventListener('wheel', onWheel, { signal: ac.signal });
+    return () => ac.abort();
+  }, [inputRef]);
```

Comment:
```suggestion
  }, []);
```

Not needed, since `inputRef` never changes. (And ESLint doesn't complain.)